### PR TITLE
Jt 37. Add choice theme report pdf

### PR DESCRIPTION
**Pull request by saffchen**

### CODE REVIEW

File: report/src/main/java/saffchen/Main.java [hunk 1]
```diff
@@ -0,0 +1,9 @@
+package saffchen;
+
+import saffchen.reports.GenerateReportCommand;
+
+public class Main {
```
Comment (pelmegov) [inline report/src/main/java/saffchen/Main.java:5]: Can it be spring application?

File: report/src/main/java/saffchen/reports/Theme.java [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+import com.itextpdf.text.BaseColor;
+import lombok.NoArgsConstructor;
+
+/**
+ * @author saffchen created on 21.08.2022
+ */
+@NoArgsConstructor
+public class Theme {
+    public BaseColor darkTheme() {
```
Comment (pelmegov) [inline report/src/main/java/saffchen/reports/Theme.java:11]: It's better to make darkThere and lightTheme subclasses with method getBaseColor, so to be able to hide some specific theme related information.

File: report/src/main/java/saffchen/reports/SwitchTheme.java [hunk 3]
```diff
@@ -0,0 +1,25 @@
+package saffchen.reports;
+
+import com.itextpdf.text.BaseColor;
+
+/**
+ * @author saffchen created on 21.08.2022
+ */
+public class SwitchTheme {
```
Comment (pelmegov) [inline report/src/main/java/saffchen/reports/SwitchTheme.java:8]: What is a responsibility of this class? It doesn't switch anything.

File: report/src/main/java/saffchen/reports/PDFReportFromFile.java [hunk 4]
```diff
@@ -58,25 +66,32 @@ private PdfPCell drawCell(String text, BaseColor fontColor, Font style) {
...
 
-            PdfPTable table = drawTable(7);
+            PdfPTable table = drawTable();
             List<String> headers = fileStorageUtils.getHeadersFromCSV();
             for (String cell : headers) {
-                table.addCell(drawCell(cell.toUpperCase(), BaseColor.LIGHT_GRAY, tableHeader));
+                table.addCell(drawCell(cell.toUpperCase(), color, tableHeader));
             }
 
+<<<<<<< HEAD
```
Comment (pelmegov) [inline report/src/main/java/saffchen/reports/PDFReportFromFile.java:90]: Is this a conflict?

**Review by pelmegov — CHANGES_REQUESTED** [review]

File: report/src/main/java/saffchen/reports/PDFReportFromFile.java (hunk 4 above)
Comment (saffchen) [reply report/src/main/java/saffchen/reports/PDFReportFromFile.java:90]: Resolve

File: report/src/main/java/saffchen/Main.java (hunk 1 above)
Comment (saffchen) [reply report/src/main/java/saffchen/Main.java]: Forgot delete this class. Fixed

File: report/src/main/java/saffchen/reports/SwitchTheme.java (hunk 3 above)
Comment (saffchen) [reply report/src/main/java/saffchen/reports/SwitchTheme.java:8]: Fixed

File: report/src/main/java/saffchen/reports/Theme.java (hunk 2 above)
Comment (saffchen) [reply report/src/main/java/saffchen/reports/Theme.java]: Fixed

**Review by pelmegov — APPROVED** [review]

**Pull request merged**